Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -15,14 +15,14 @@ public static void Demo()
                 var age = PromptInt("What is your age");
                 var weight = PromptFloat("What is your weight in kilograms");
                 var heightCm = PromptFloat("What is your height in centimeters");
-                var BMI = CalculateBmi(weight, heightCm / 100);
+                var Bmi = CalculateBmi(weight, heightCm / 100);
```

Comment:
Being a bit picky here but keeping up with your own style is very important if you advance in programming and your projects get larger and you end up with multiple files that you want to read quickly without getting disturbed by different styles.
Change it to be all lowercase. Your local variables should all start with a lowercase letter and should only have one uppercase letter for each new word beginning in the variable name. :)